src/main/java/org/sonatype/nexus/ci/util/IqUtil.groovy

```diff
@@ -42,6 +43,12 @@ class IqUtil
     return client.getApplicationsForApplicationEvaluation()
   }
 
+  static boolean verifyOrCreateApplication(final IqClient iqClient,
```

Comment:
> Might not be worthwhile creating this in the utility class since there isn't any extra work it's doing that we can't just call on the iqClient itself.

Oh geez, you are right.  That would make things so much easier.  

I used the utility class at the beginning for iqclient.  But because of groovy mocking issues, I removed all the code.   Oh, that would have made things so much easier if I just never used the utility class.

Reply:
>Might not be worthwhile creating this in the utility class since there isn't any extra work it's doing that we can't just call on the iqClient itself.

Changed df803ed